packages/create-vite/template-vanilla-ts/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="favicon.svg" />
```

Comment:
```suggestion
    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```
Maybe change it to this and remove `template-vanilla-ts/favicon.svg`?